client/src/main/kotlin/io/provenance/classification/asset/client/client/impl/DefaultACExecutor.kt

```diff
@@ -42,21 +42,21 @@ class DefaultACExecutor(
         execute: OnboardAssetExecute<T>,
         signerAddress: String,
     ): MsgExecuteContract = querier
-        .queryAssetDefinitionByAssetType(execute.onboardAsset.assetType)
+        .queryAssetDefinitionByAssetType(execute.assetType)
```

Comment:
These new refactors make using (and declaring) the objects for the client a lot easier, as can be seen by the flat structure here